vmdb/app/apis/miqservices_ops.rb

```diff
@@ -244,6 +243,7 @@ def agent_config(hostId, config)
 
   def agent_register(xmlFile)
     ret = false
+    require 'miq-xml'
```

Comment:
@blomquisg Yes, I'm not **happy** about it.  miq-xml loads rexml, nokogiri, our patches, so it's not tiny.  This is definitely not the **right** solution but the problem is we have enormous files that have only one or two methods that need miq-xml so the intention of the require is not clear.  We should probably have miq-xml uses moved outside of these classes and into a single module/class that provides the miq-xml require and use cases.

Similar to this:  http://twin.github.io/2015/01/require-only-what-you-require/

I'm fine with just fixing the problem at hand: ensure we require miq-xml in files where it's needed.

Though, I think seeing the :poop: of this PR demonstrates the problem.